search/elasticSearch.js

```diff
@@ -15,7 +15,7 @@ class ElSearch {
    */
   constructor() {
     this.client = new elasticsearch.Client({
-      host: 'localhost:9200'
+      host: '52.52.22.4:9200'
```

Comment:
Maybe we should remove all hostnames and ports to one of the config files and export them to the process env?